.github/workflows/build.yaml

```diff
@@ -0,0 +1,73 @@
+name: build
+
+on:
+  push:
+    branches:
+      - "branch-*"
+    tags:
+      - v[0-9][0-9].[0-9][0-9].[0-9][0-9]
+  workflow_dispatch:
+    inputs:
+      branch:
+        required: true
+        type: string
+      date:
+        required: true
+        type: string
+      sha:
+        required: true
+        type: string
+      build_type:
+        type: string
+        default: nightly
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}-${{ github.event_name }}
+  cancel-in-progress: true
+
+jobs:
+  cpp-build:
+    secrets: inherit
+    uses: rapidsai/shared-workflows/.github/workflows/conda-cpp-build.yaml@branch-24.12
+    with:
+      build_type: ${{ inputs.build_type || 'branch' }}
+      branch: ${{ inputs.branch }}
+      date: ${{ inputs.date }}
+      sha: ${{ inputs.sha }}
+      node_type: cpu32
```

Comment:
Do you think we still need `cpu32` here? Default is `cpu16`, I think. We use `cpu32` for cugraph because it's quite large with sccache misses. Are the GNN packages equally large? I would guess they are faster to compile than cugraph but don't know for sure.
```suggestion
```

Reply:
That's an excellent point! I think you're right that this doesn't need `cpu32`. `conda-build-cpp` is just building `libwholegraph`, and it accepts the default runner when built over in the `wholegraph` repo: https://github.com/rapidsai/wholegraph/blob/6821f7ad99b4418ea59887249720469fa676bad2/.github/workflows/build.yaml#L29-L37

Reply:
Just pushed https://github.com/rapidsai/cugraph-gnn/pull/61/commits/805062d33f7013865870992ec3956c548a847dff changing these builds and those on PRs to use whatever the default runner size is for the build workflows.